plugins/log-loader/index.ts

```diff
@@ -53,4 +49,17 @@ export const activate = (api: BrimApi) => {
   })
 }
 
+function nodeJSReadableStreamFromReadableStream(
```

Comment:
Style-wise, should this anonymous (`const nodeJSReadableStreamFromReadableStream = ...`) rather than named?

Reply:
Also, I'd like to acknowledge that this is ugly. If there's a way to avoid it that I missed, let me know!

Reply:
Nope, named is still cool.